WordPress/WordPressDraftActionExtension/Base.lproj/MainInterface.storyboard

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this file should be directly in ` WordPress/WordPressDraftActionExtension/` since we don't localize storyboards (we are doing that on the Share extension), what do you think?

Reply:
Sounds good @elibud .... changes coming!